pages/index.tsx

```diff
@@ -95,199 +100,136 @@ function IndexPage({ articles }: Props) {
-            </Text>
-          </EnergyFactContainer>
-          <EnergyFactContainer backgroundImage="/night-sky-2-card.png">
-            <Heading
-              as="h3"
-              fontFamily="Lora, serif"
-              fontSize="1.5rem"
-              fontWeight="500"
-              color="rgba(255,255,255,0.6)"
-            >
-              $90M
-            </Heading>
-            <Text color="white">
-              The energy we consume to heat, cool and power our homes represents
-              14% of Kingston’s greenhouse gas emissions and cost nearly $90
-              million annually.
-            </Text>
-          </EnergyFactContainer>
-        </SimpleGrid>
-        <SimpleGrid columns={{ md: 1, lg: 2 }} spacing={2} marginTop={2}>
-          <EnergyFactContainer backgroundImage="/night-sky-1-card.png">
-            <Heading
-              as="h3"
-              fontFamily="Lora, serif"
-              fontSize="1.5rem"
-              fontWeight="500"
-              color="rgba(255,255,255,0.6)"
-            >
-              -30%
-            </Heading>
-            <Text color="white">
-              Kingston was the first Ontario municipality to declare a climate
-              emergency, setting a goal to cut greenhouse gas emissions 30% by
-              the year 2030.
-            </Text>
-          </EnergyFactContainer>
-          <EnergyFactContainer backgroundImage="/flower-card.png">
-            <Heading
-              as="h3"
-              fontFamily="Lora, serif"
-              color="rgba(0,0,0,0.6)"
-              fontSize="1.5rem"
-              fontWeight="500"
-            >
-              Committing to change
-            </Heading>
-            <Text>
-              Greenhouse gases such as CO2, which are created from home heating
-              systems, are contributing to the current climate crisis. You can
-              help minimize your climate impact by evaluating your house to
-              eliminate substandard conditions.
-            </Text>
-          </EnergyFactContainer>
+        <Subheading>{metadata.whyThisToolTitle}</Subheading>
+        <SimpleGrid columns={{ base: 2, lg: 2 }} spacing={2} marginTop="16pt">
+          {facts.map((fact, index) => {
+            return (
+              <EnergyFactContainer
+                key={fact.title}
+                backgroundImage={fact.backgroundImage}
+              >
+                <Heading
+                  as="h3"
+                  fontFamily="Lora, serif"
+                  fontSize="1.5rem"
+                  fontWeight="500"
+                  color={
+                    fact.lightTextColor ? "rgba(255,255,255,0.6)" : "black"
+                  }
+                >
+                  {fact.title}
+                </Heading>
+                <Text
+                  width={index === 0 ? "70%" : "100%"}
+                  // better way to make width dynamic?
+                  color={fact.lightTextColor ? "white" : "black"}
```

Comment:
Similar idea with the other elements. Chakra uses an off-black color for text. We should preserve that design.
```suggestion
                  color={fact.lightTextColor ? "white" : undefined}
```